pkg/cassandra/config.go

```diff
@@ -267,6 +272,7 @@ func filterConfigForVersion(cassandraVersion string, filteredConfig *api.Cassand
 		filteredConfig.CassandraYaml.RequestScheduler = nil
 		filteredConfig.CassandraYaml.RequestSchedulerId = nil
 		filteredConfig.CassandraYaml.RequestSchedulerOptions = nil
+		filteredConfig.CassandraYaml.StartRpc = nil
```

Comment:
Is it equivalent to `false` ? 🤔

Reply:
It is nil, which I think means for Cassandra 4.0 it will not be included in the YAML. This is desirable because Cassandra 4.0 does not allow this parameter.